subsys/net/lib/nrf_provisioning/src/nrf_provisioning.c

```diff
@@ -241,12 +257,6 @@ static int settings_init(void)
 		return err;
 	}
 
-	err = settings_load_subtree(settings.name);
```

Comment:
Why is this removed?
If the application does not do `settings_load()` after this is initialized, then we end up not loading anything from settings store.